### PR TITLE
Add homepage header to allow user to log in and switch accounts

### LINKED ISSUE #241: Add link to dashboard from homepage header
Many users dont find a way to access dashboard, we should have a link on the homepage perhaps with Sign up | Log in.

### PR DESCRIPTION
**Pull request by gregory-latinier**
User not connected : add login button
User connected: add the menu account
Replace the selected language by an icon
Fix #241

### CODE REVIEW

File: src/styles/common.less [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import "~bootstrap/dist/css/bootstrap.min.css";
 @import "~antd/dist/antd.less";
 @import "./custom.less";
+@import '//at.alicdn.com/t/font_317125_wqz6u4c8idi8jjor.css';
```
Comment (bonustrack) [inline src/styles/common.less]: Can we have the css within the app? Not as external ressource

File: .env.example [hunk 2]
```diff
@@ -9,6 +9,6 @@ CSP_DEFAULT='self'
 CSP_SCRIPT_SRC='self','unsafe-eval','unsafe-inline'
 CSP_CONNECT_SRC='self',*.steemit.com
 CSP_FRAME_SRC='self'
-CSP_STYLE_SRC='self','unsafe-inline'
+CSP_STYLE_SRC='self','unsafe-inline',*.alicdn.com
 CSP_IMG_SRC='self',steemitimages.com,steemit-production-imageproxy-thumbnail.s3.amazonaws.com,data:
-CSP_FONT_SRC='self'
+CSP_FONT_SRC='self',*.alicdn.com,data:
```
Comment (bonustrack) [inline .env.example]: I guess the 2 x aliccdn.com are not necessary anymore, but `data:` yes, can you update that?

**Comment by bonustrack** [comment]
- Language icon should have `font-weight: 500;`, now it look too much bold compared to Paresh design.
- The dashboard (when logged) should also have SteemConnect logo and language selector
- SteemConnect logo should have a link to `/`
- Header on homepage should have `margin-top: 10px;` (now we have 40px it's too much)
- Header must be full-width like this on both homepage and dashboard:
image

**Comment by gregory-latinier** [comment]
I can't get the language dropdown to work for the dashboard without refactoring the project.
Since it will be done when the new design will be validated I'll let this bug for now and solve it later

**Pull request closed without merging**